src/main/kotlin/no/nav/syfo/sykmelding/KafkaSykmeldingConsumer.kt

```diff
@@ -115,7 +117,14 @@ class KafkaSykmeldingConsumer(
         val arbeidstakerPersonident = PersonIdent(receivedSykmeldingDTO.personNrPasient)
         val hasExistingUbehandlet = connection.getPersonOppgaverByReferanseUuid(referanseUuid)
             .any { it.behandletTidspunkt == null }
-        if (!hasExistingUbehandlet) {
+        val hasExistingDuplicate = sykmeldingFieldsRepository.findExistingPersonoppgaveFromSykmeldingFields(
```

Comment:
Denne finner bare en tidligere forekomst med samme verdier i *alle* de tre feltene. Bør kanskje forbedres slik at det også blir regnet som en duplikat dersom verdien i ett eller to felt er de samme og det/de som ikke er det samme er tomt i den nye sykmeldingen.